### PR TITLE
adds Generic LR1121 targets

### PR DESCRIPTION
**Pull request by JyeSmith**
https://github.com/ExpressLRS/ExpressLRS/pull/2540

Lets get this in ASAP for easy Configurator flashing.

Thanks.

### CODE REVIEW

File: TX/Generic LR1121 Gemini.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "led_rgb_isgrb": true,
+    "ledidx_rgb_status": [0],
+    "ledidx_rgb_boot": [0],
+
+    "radio_dcdc": true,
+
+    "use_backpack": false,
+    "debug_backpack_baud": 460800,
+    "debug_backpack_rx": 3,
+    "debug_backpack_tx": 1,
```
Comment (pkendall64) [inline TX/Generic LR1121 Gemini.json:37]: Probably don't want to use the main port for backpack, especially as you've used it for the s.port above.

File: TX/Generic LR1121 Gemini.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+    "serial_rx": 1,
+    "serial_tx": 1,
```
Comment (pkendall64) [inline TX/Generic LR1121 Gemini.json:3]: For a generic target I don't think we want to be using the main serial port for this do we?

File: TX/Generic LR1121 Gemini.json (hunk 1 above)
Comment (JyeSmith) [reply TX/Generic LR1121 Gemini.json]: Fixed.

Yep, testing left overs.

File: TX/Generic LR1121 Gemini.json (hunk 2 above)
Comment (JyeSmith) [reply TX/Generic LR1121 Gemini.json]: Fixed to match dev board.

**Review by pkendall64 — APPROVED** [review]

**Pull request merged**